kolena/_utils/state.py

```diff
@@ -142,7 +142,7 @@ def get_token(
     base_url: Optional[str] = None,
     proxies: Optional[Dict[str, str]] = None,
 ) -> API.ValidateResponse:
-    base_url = base_url or get_client_base_url()
+    base_url = base_url or get_client_state().base_url
```

Comment:
After fix, the only direct invocation of `get_client_base_url` outside of initializing client_state is in `kolena_session`, which is intended usage.

Reply:
Thoughts on removing `get_client_base_url` so as to not encourage other misusage down the line?

Reply:
Marked it as private as an alternative, `_get_api_base_url`, since I think it serves a purpose to avoid remembering env variable name.